daemon/cluster/executor/container/controller.go

```diff
@@ -183,6 +183,10 @@ func (r *controller) Start(ctx context.Context) error {
 
 	// no health check
 	if ctnr.Config == nil || ctnr.Config.Healthcheck == nil {
+		if err := r.adapter.backend.ActivateContainerServiceBinding(r.adapter.container.name()); err != nil {
```

Comment:
Make a method on adapter to capture all this. Should just be `r.adapter.activateServiceBinding()`.